tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "include": ["src/**/*", "test/**/*", "types/**/*"],
+  "include": ["src/**/*", "test/**/*", "types/**/*", "atqa-framework/**/*"],
```

Comment:
You need to add a reference to `atqa-framework` here and put that path to include there, not here